### ISSUE TITLE
Catalina support

### ISSUE THREAD

**Issue by Jolopu** [jlj/NativeDisplayBrightness#3]
It used to work fine with Mojave but stopped to work on Catalina.

**Comment by jlj** [comment]
Works on without issue on Catalina on my configuration.

@Jolopu Can you check that your privacy settings enable Accessibility for *NativeDisplayBrightness*?

![Screenshot 2020-03-19 at 10 07 50](https://user-images.githubusercontent.com/1850262/77050581-462ebd80-69ca-11ea-8432-c0833f59f289.png)

**Comment by Jolopu** [comment]
@jlj Yes, I checked the privacy settings and I can see the process running but it didn't do anything. I tried this fork:
https://github.com/the0neyouseek/MonitorControl
For some reason it works.

**Comment by jlj** [comment]
Strange… To double-check under Catalina 10.15.4, I removed NativeDisplayBrigthness from my *Privacy* settings as well as from *Users and Groups* Login items, deleted the app, restarted the mac and reinstalled the app (v0.0.8 from this repo).

After giving permission to enable accessibility, everything works as expected. 🤔

@Jolopu  Can you do the same operations on your system and post the result?

**Comment by Jolopu** [comment]
I run Catalina 10.15.3. I followed your steps and here is what happened:
Privacy settings are enabled, login item is enabled. System rebooted. Process is running.  Icon not showing in toolbar. Brightness keys show no effect.

![Bildschirmfoto 2020-03-19 um 14 56 36](https://user-images.githubusercontent.com/34797349/77075666-f9131180-69f2-11ea-88e6-8ca3771a6bcb.png)
![Bildschirmfoto 2020-03-19 um 14 54 15](https://user-images.githubusercontent.com/34797349/77075668-f9aba800-69f2-11ea-8b7c-0fd0a9d339dc.png)
![Bildschirmfoto 2020-03-19 um 14 57 39](https://user-images.githubusercontent.com/34797349/77075669-fa443e80-69f2-11ea-8d8c-d19f73e13819.png)

Unistalled NativeDisplayBrightness, installed MonitorControl and it works again:

![Bildschirmfoto 2020-03-19 um 15 24 40](https://user-images.githubusercontent.com/34797349/77077641-c74f7a00-69f5-11ea-8b31-e662afbb5228.png)

**Comment by jlj** [comment]
@Jolopu Thanks for doing this. 😃

*Btw, NativeDisplayBrightness does not display an icon in the Mac menu toolbar, so this part is normal. For the rest, I will investigate on other machines.*

**Comment by Jolopu** [comment]
@jlj You are welcome. 😃 I hope you find the cause. Maybe it has something to do with my system and apps installed. I had the exact same issue with this fork before:
https://github.com/Bensge/NativeDisplayBrightness/issues/20

**Comment by psykocar** [comment]
Same problem with MacOs 10.15.4 (MBP 15" retina early 2013). I can see the native osd graphics by pressing fn+F1/F2 but nothing happens. Built and reinstalled several times. I also tried MonitorControl and it is not working either. Please let me know if you find out something.
